pam/gdm/debug.go

```diff
@@ -0,0 +1,8 @@
+//go:build pam_gdm_debug
+
+package gdm
+
+func init() {
+	checkMembers = checkMembersReal
+	validateJSON = validateJSONReal
+}
```

Comment:
Oh, this is weird since I did test this, but before the renaming, so something could have happened at later point. :thinking:

Reply:
Ahhhhhhhhhhhhh... No wait, this is expected :-D

It's not yet used because pam module isn't depending on gdm (yet), it will in next branches. So for now the code isn't used but it will.

In fact just add a `_, _ = gdm.NewBinaryJSONProtoRequest(nil)` inside `pam.go` and this works fine.

Reply:
Ok, renamed...

Indeed the code is initialized now as evidence of:

```diff
diff --git a/pam/gdm/debug.go b/pam/gdm/debug.go
index 367203e..ebbee37 100644
--- a/pam/gdm/debug.go
+++ b/pam/gdm/debug.go
@@ -5,4 +5,6 @@ package gdm
 func init() {
 	checkMembersFunc = checkMembersDebug
 	validateJSONFunc = validateJSONDebug
+
+	panic("Hello debug!")
 }
diff --git a/pam/pam.go b/pam/pam.go
index 3d71b49..508048b 100644
--- a/pam/pam.go
+++ b/pam/pam.go
@@ -16,6 +16,7 @@ import (
 	"github.com/ubuntu/authd"
 	"github.com/ubuntu/authd/internal/consts"
 	"github.com/ubuntu/authd/internal/log"
+	"github.com/ubuntu/authd/pam/gdm"
 	"golang.org/x/term"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
@@ -84,6 +85,8 @@ func (h *pamModule) Authenticate(mTx pam.ModuleTransaction, flags pam.Flags, arg
 		interactiveTerminal: interactiveTerminal,
 	}
 
+	_, _ = gdm.NewBinaryJSONProtoRequest(nil)
+
 	if err := mTx.SetData(authenticationBrokerIDKey, nil); err != nil {
 		return err
 	}
```